java/spring-framework/pom.xml

```diff
@@ -0,0 +1,40 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>benchmark.spring-framework</groupId>
+    <artifactId>benchmark</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>benchmark</name>
+    <description>Demo project for Spring Framework</description>
+    <packaging>war</packaging>
+
+    <parent>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-framework-bom</artifactId>
+        <version>5.2.0.RELEASE</version>
+    </parent>
+    <properties>
+        <java.version>1.8</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.10.1</version>
```

Comment:
could you remove this ? 
it's add more maintenance (e.g : we do not care about which version of each dependency, but we care about version of **frameworks** in use)

Reply:
It is necessary to specify this dependency because it is required for the /user/{id} endpoint, but spring marks it as optional.

Reply:
in fact I mean specify the version of dependency, not dependency itself

Reply:
Sorry let me make sure I understand you: you mean the version should be extracted like a property? e.g. <version>${jackson-databind.version}</version>

Reply:
nop
I mean, this version tag is not very much required, in my opinion.
Because :
+ it adds more maintenance / updates
+ it could lead to mistaken result in case of this is outdated (in the future)

Reply:
I'm confused. The dependency to some version of jackson-databind is mandatory. Then to specify it in maven, there is no other way than adding the dependency and the version because spring framework lists it as optional.
See https://github.com/spring-projects/spring-framework/blob/21d751d22202c6df2d0144b155979c37b45fb167/build.gradle#L46

However we can stick to 2.10.x or even 2.x by using the [2.10,2.11) notation.

Reply:
Yeah.

This is not ideal, but helps a lot :heart: 

Could you check for the same thing on all `java` **frameworks** `here` ?